airbyte-server/src/main/java/io/airbyte/server/apis/ConfigurationApi.java

```diff
@@ -385,6 +391,18 @@ public WbConnectionRead webBackendGetConnection(@Valid ConnectionIdRequestBody c
     return execute(() -> webBackendConnectionsHandler.webBackendGetConnection(connectionIdRequestBody));
   }
 
+  // MIGRATION
```

Comment:
We should really remove any mention of "migration" it it just a config import/export